<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TipoDeCambio</class>
 <widget class="QMainWindow" name="TipoDeCambio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tipo de cambio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblConvertir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Convertir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leImporte">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbDe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>De:</string>
    </property>
    <widget class="QRadioButton" name="rbDeUSD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>USD</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbDeEUR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>EUR</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbDePEN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PEN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbDeGBP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>GBP</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbA">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>71</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>A:</string>
    </property>
    <widget class="QRadioButton" name="rbAUSD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>USD</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbAEUR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>EUR</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbAPEN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PEN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbAGBP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>GBP</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblResultado">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCambio">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTipoCambio">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convertir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>327</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
